ddtrace/appsec/_iast/_taint_tracking/Initializer/Initializer.cpp

```diff
@@ -206,13 +264,15 @@ Initializer::create_context()
     // Create a new taint_map
     auto map_ptr = create_tainting_map();
     ThreadContextCache.tx_map = map_ptr;
+    import_symbols_into_cache();
 }
 
 void
 Initializer::reset_context()
 {
     clear_tainting_maps();
     ThreadContextCache.tx_map = nullptr;
+    imported_cache.clear();
```

Comment:
Do we really want to clear the cache on context reset?

Reply:
Yup. Segmentation fault if we dont :)

Reply:
Can't we call it only in `~Initializer()`? Don't we call this in between requests?